packages/deprecation-crawler/sandbox/package.json

```diff
@@ -1,4 +1,7 @@
 {
+  "version1": "deprecation-crawler@0.0.0-sandbox",
+  "version2": "0.0.0-sandbox",
```

Comment:
What are these for? 👀

Reply:
removed it. I wanted to document it somewhere, but its already in the code as comments and I anyway need to write a proper docs document fror that